### PR TITLE
feat : Added Alert Section

### PR DESCRIPTION
**Pull request by Omkar-Chaskar**
Added Alert boxes and style

### CODE REVIEW

**Review by Raghav888 — APPROVED** [review]
Approved

File: Components/alert.css [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  .alert p span{
+    position: relative;
+    float: right;
+    top: 3px;
+    cursor: pointer;
+  }
+  
+  .alert-primary {
+    color: #021777;
+    background-color: #7e95fc;
```
Comment (Bhakti199) [inline Components/alert.css:21]: use color variables here

File: Components/alert.css [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  }
+  
+  .alert-success {
+    color: #046611;
+    background-color: #94d4a2;
+  }
+  
+  .alert-danger {
+    color: #9c0505;
+    background-color: #d49999;
```
Comment (Bhakti199) [inline Components/alert.css:36]: here too.

**Review by Bhakti199 — APPROVED** [review]
good to merge, suggestion would be - use color variables

File: Components/alert.css (hunk 2 above)
Comment (Omkar-Chaskar) [reply Components/alert.css:36]: I will change it in next PR . Thank You

**Pull request merged**